CHANGELOG.md

```diff
@@ -16,7 +16,8 @@ All user visible changes to this project will be documented in this file. This p
 - UI:
     - Chat page:
         - Inability to navigate back after clicking on direct link. ([#1504], [#1460])
-
+    - Chats tab:
+        - Redesign UnreadCounter to display numbers up to 999.999 ([#1507])
```

Comment:
End users don't know what an `UnreadCounter` is, they aren't developers. And this changelog is what is published to those users who simply use the application, not the developers.